app/models/parent.rb

```diff
@@ -45,6 +45,7 @@ class Parent < ApplicationRecord
             length: {
               maximum: 300
             }
+  validates :contact_method_other, presence: true, if: :contact_method_other?
```

Comment:
This isn't confusing at all. 😵  Not your fault, but this line does highlight the unfortunate naming we have of the `contact_method` enum.

Reply:
mm yeah not ideal.